### PR TITLE
Ref Guide - OpenSCAP Overview

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10588

Moves content to new chapter in Admin Guide, then references it.

New chapter will require editing and updated screenshots, (will do in new PR). It's here: https://github.com/uyuni-project/uyuni-docs/pull/89

### CODE REVIEW

File: modules/administration/pages/openscap.adoc [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+It is a line of specifications maintained by the National Institute of Standards and Technology (NIST) for maintaining system security for enterprise systems.
+
+{productname} uses OpenSCAP to implement the SCAP specifications.
+OpenSCAP is an auditing tool that utilizes the Extensible Configuration Checklist Description Format (XCCDF). XCCDF is a standard way of expressing checklist content and defines security checklists.
+It also combines with other specifications such as Common Platform Enumeration (CPE), Common Configuration Enumeration (CCE), and Open Vulnerability and Assessment Language (OVAL), to create a SCAP-expressed checklist that can be processed by SCAP-validated products.
+
+[[s1-openscap-features]]
+== OpenSCAP Features
+
+
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+A tool to verify a system confirms to a standard::
+{productname} uses OpenSCAP as an auditing feature.
+It allows you to schedule and view compliance scans for any system.
+
+SCAP content::
+SCAP content files defining the test rules can be created from scratch if you understand at least XCCDF or OVAL.
+XCCDF content is also frequently published online under open source licenses and this content can be customized to suit your needs.
++
+The [systemitem]``openscap-content``
+package provides default content guidance for systems via a template.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
The [systemitem]``openscap-content`` package provides default content guidance for systems via a template.
```

File: modules/administration/pages/openscap.adoc [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+A tool to verify a system confirms to a standard::
+{productname} uses OpenSCAP as an auditing feature.
+It allows you to schedule and view compliance scans for any system.
+
+SCAP content::
+SCAP content files defining the test rules can be created from scratch if you understand at least XCCDF or OVAL.
+XCCDF content is also frequently published online under open source licenses and this content can be customized to suit your needs.
++
+The [systemitem]``openscap-content``
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+package provides default content guidance for systems via a template.
+
+
+[NOTE]
+====
+{suse}
+supports the use of templates to evaluate your systems.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
{suse} supports the use of templates to evaluate your systems.
```

File: modules/administration/pages/openscap.adoc [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+SCAP content files defining the test rules can be created from scratch if you understand at least XCCDF or OVAL.
+XCCDF content is also frequently published online under open source licenses and this content can be customized to suit your needs.
++
+The [systemitem]``openscap-content``
+package provides default content guidance for systems via a template.
+
+
+[NOTE]
+====
+{suse}
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+#
+ ke 2013-08-28: Do we have SCAP content providers? Such as: The United States Government
+   Configuration Baseline (USGCB) for RHEL5 Desktop or Community-provided content (openscap-content
+   package)? For more info, see
+   https://access.redhat.com/site/documentation/en-US/Red_Hat_Network_Satellite/5.5/html/User_Guide/chap-Red_Hat_Network_Satellite-User_Guide-OpenSCAP.html #
+endif::showremarks[]
+
+
+SCAP was created to provide a standardized approach to maintaining system security, and the standards that are used will therefore continually change to meet the needs of the community and enterprise businesses.
+New specifications are governed by NIST's SCAP Release cycle in order to provide a consistent and repeatable revision work flow.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
New specifications are governed by NIST's SCAP Release cycle to provide a consistent and repeatable revision work flow.
```

File: modules/administration/pages/openscap.adoc [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+For more information, see http://scap.nist.gov/timeline.html.
+
+[[s1-openscap-suma-prerq]]
+== Prerequisites for Using OpenSCAP in {productname}
+
+
+The following sections describe the server and client prerequisites for using OpenSCAP.
+
+Package Requirements::
+As Server: {productname}
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
As Server: {productname} 1.7 or later.
```

File: modules/administration/pages/openscap.adoc [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+1.7 or later.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+== Performing Audit Scans
+
+
+OpenSCAP integration in {productname} provides the ability to perform audit scans on client systems.
+This section describes the available scanning methods.
+
+.OpenSCAP Scans via Salt ssh-push Clients
+[IMPORTANT]
+====
+Currently performing an OpenSCAP scan is disabled in the WebUI for Salt ssh-push clients.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
Currently performing an OpenSCAP scan is disabled in the {webui} for Salt ssh-push clients.
```

File: modules/administration/pages/openscap.adoc [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+This functionality will be adapted and enabled in a future release.
+====
+
+[[pro-os-suma-audit-scans-webui]]
+.Procedure: Scans via the Web Interface
+. To perform a scan via the Web interface, log in to {productname} .
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
. To perform a scan via the Web interface, log in to {productname}.
```

File: modules/administration/pages/openscap.adoc [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+. Click on menu:Systems[] and select the target system.
+. Click on menu:Audit[Schedule] .
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
. Click on menu:Audit[Schedule].
```

File: modules/administration/pages/openscap.adoc [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+. Fill in the Schedule New XCCDF Scan form. For more information about the fields on this page, see:
+xref:reference:audit/system-details-audit.adoc#sm-audit-schedule[Schedule Audit]
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+Currently performing an OpenSCAP scan is disabled in the WebUI for Salt ssh-push clients.
+This functionality will be adapted and enabled in a future release.
+====
+
+[[pro-os-suma-audit-scans-webui]]
+.Procedure: Scans via the Web Interface
+. To perform a scan via the Web interface, log in to {productname} .
+. Click on menu:Systems[] and select the target system.
+. Click on menu:Audit[Schedule] .
+. Fill in the Schedule New XCCDF Scan form. For more information about the fields on this page, see:
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
. Fill in the Schedule New XCCDF Scan form. For more information about the fields on this page, see xref:reference:audit/system-details-audit.adoc#sm-audit-schedule[].

```

File: modules/administration/pages/openscap.adoc [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+
+or
+
+----
+service osad start
+----
++
+
+
+
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
// FIXME: this xref will only work if the target is in the same file (or module?).
```

File: modules/administration/pages/openscap.adoc [hunk 15]
```diff
@@ -0,0 +1,228 @@
...
+----
+service osad start
+----
+
+
+.Enabling Upload of Detailed SCAP Files
+[NOTE]
+====
+To make sure detailed information about the scan will be available, activate the upload of detailed SCAP files on the clients to be evaluated.
+On the menu:Admin[]
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
Navigate to menu:Admin[Organization] and select an organization.
```

File: modules/administration/pages/openscap.adoc [hunk 16]
```diff
@@ -0,0 +1,228 @@
...
+ page, click on menu:Organization[]
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
 ```

File: modules/administration/pages/openscap.adoc [hunk 17]
```diff
@@ -0,0 +1,228 @@
...
+ and select one.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 18]
```diff
@@ -0,0 +1,228 @@
...
+Click on the menu:Configuration[]
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
On the menu:Configuration[] tab, check [guimenu]``Enable Upload Of Detailed SCAP Files``.
```

File: modules/administration/pages/openscap.adoc [hunk 19]
```diff
@@ -0,0 +1,228 @@
...
+ tab and check menu:Enable Upload Of Detailed SCAP Files[]
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 20]
```diff
@@ -0,0 +1,228 @@
...
+.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 21]
```diff
@@ -0,0 +1,228 @@
...
+This feature generates an additional HTML version when you run a scan.
+The results will show an extra line like: ``Detailed Results: xccdf-report.html xccdf-results.xml
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]: Probably also not ok...
```suggestion
The results will show an extra line like:
+
----
Detailed Results: xccdf-report.html xccdf-results.xml scap-yast2sec-oval.xml.result.xml
----
```

File: modules/administration/pages/openscap.adoc [hunk 22]
```diff
@@ -0,0 +1,228 @@
...
+     scap-yast2sec-oval.xml.result.xml``.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
 ```

File: modules/administration/pages/openscap.adoc [hunk 23]
```diff
@@ -0,0 +1,228 @@
...
+====
+
+[[s2-openscap-suma-audit-scans_view]]
+== Viewing SCAP Results
+
+
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

File: modules/administration/pages/openscap.adoc [hunk 24]
```diff
@@ -0,0 +1,228 @@
...
+There are three methods of viewing the results of finished scans:
+
+* Via the Web interface. Once the scan has finished, the results should show up on the menu:Audit[] tab of a specific system. This page is discussed in <<s2-openscap-suma-pages>>.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
* Via the Web interface.
When the scan has finished, results show up on the menu:Audit[] tab of a specific system.
This page is discussed in <<s2-openscap-suma-pages>>.
// ^ FIXME: xref
```

File: modules/administration/pages/openscap.adoc [hunk 25]
```diff
@@ -0,0 +1,228 @@
...
+[[s2-openscap-suma-pages]]
+== OpenSCAP {productname} Web Interface
+
+
+The following sections describe the tabs in the {productname} {webui} that provide access to OpenSCAP and its features.
+
+=== OpenSCAP Scans Page
+
+
+Click the menu:Audit[]
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
Navigate to menu:Audit[OpenSCAP].
```

File: modules/administration/pages/openscap.adoc [hunk 26]
```diff
@@ -0,0 +1,228 @@
...
+ tab on the top navigation bar, then OpenSCAP on the left.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc]:
```suggestion
```

**Comment by Loquacity** [comment]
I'll do the edits in a new PR.

**Pull request merged**